core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2076,4 +2076,15 @@ private CarbonCommonConstants() {
    */
   public static final String CARBON_QUERY_DATAMAP_BLOOM_CACHE_SIZE_DEFAULT_VAL = "512";
 
+  public static final String CARBON_LOCAL_DICTIONARY_MAX_THRESHOLD =
```

Comment:
add a comment currently it is internal

Reply:
I think we should optimize this variable name.
The first time I saw this I thought it was duplicated with another threshold for local dictionary. One is number based, another is storage size based. Please take care of the readability.

Reply:
changed to CARBON_LOCAL_DICTIONARY_MAX_SIZE_THRESHOLD

Reply:
It still failed to make it clear that what kind of size it supposed to be since we have a storage size and a counting size.